schema.graphql

```diff
@@ -113,8 +113,10 @@ type Token @entity {
 type TokenBalance @entity {
   id: ID! # account address + token address
   owner: Account!
-  balance: BigInt!
-  value: BigInt!
+  balanceAmount: BigInt!
+  balanceValue: BigInt!
+  totalValue: BigInt!
+  tokenPrice: BigInt!
```

Comment:
Can we move the price into the `Token` instead?

Also, can we add `totalAmount`?